templates/spring.rb

```diff
@@ -1,6 +0,0 @@
-Spring.watch(
```

Comment:
Does Rails add this automatically? Where's that come from?

Reply:
Yes, Rails automatically adds a spring initializer that is exactly the same as this one. However, AFAIK Rails will remove Spring from the default `rails new` template.